movie-smt-ui/src/components/MovieCard/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import styles from "./MovieCard.module.scss";
+import { Movie } from "../../utils/types";
+import {
+  Card,
+  CardActionArea,
+  CardContent,
+  CardMedia,
+  createTheme,
+  Skeleton,
+  Typography,
+} from "@mui/material";
+import { ThemeProvider } from "@emotion/react";
+
+const theme = createTheme({
+  palette: {
+    background: {
+      paper: "#242424",
+    },
+    text: {
+      primary: "#CFD3FD",
+      secondary: "#46505A",
+    },
+    action: {
+      active: "#001E3C",
+    },
+  },
+});
+
+export function MovieCard({ id, title, overview, poster_path, adult, genres, release_date, vote_average, spoken_languages, runtime, status, tagline}: Movie) {
```

Comment:
We should keep all component package names upper case like this one to keep them consistent